pkgs/os-specific/linux/tuxedo-drivers/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  kernel,
+  linuxHeaders,
+  pahole,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tuxedo-drivers-${kernel.version}";
+  version = "4.7.0";
+
+  src = fetchFromGitLab {
+    group = "tuxedocomputers";
+    owner = "development/packages";
+    repo = "tuxedo-drivers";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-wZUQHIkbxt9ckTFs8VTrA5I+ebBeaOm+Fb0+GqX5y0c=";
+  };
+
+  buildInputs = [
+    pahole
+    linuxHeaders
+  ];
+
+  makeFlags = [ "KDIR=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build" ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out/lib/modules/${kernel.modDirVersion}"
+
+    find src/ -type f -name '*.ko' \
+      -exec mv {} $out/lib/modules/${kernel.modDirVersion} \;
+
+    runHook postInstall
+  '';
+
+  meta = {
+    broken = stdenv.hostPlatform.isAarch64 || (lib.versionOlder kernel.version "5.5");
+    description = "Keyboard and hardware I/O driver for TUXEDO Computers laptops";
+    homepage = "https://gitlab.com/tuxedocomputers/development/packages/tuxedo-drivers";
+    license = lib.licenses.gpl3Plus;
+    longDescription = ''
+      Drivers for several platform devices for TUXEDO notebooks:
+      - Driver for Fn-keys
+      - SysFS control of brightness/color/mode for most TUXEDO keyboards
+      - Hardware I/O driver for TUXEDO Control Center
+
+      Can be used with the "hardware.tuxedo-drivers" NixOS module.
+    '';
+    maintainers = [ lib.maintainers.blanky0230 ];
```

Comment:
@Keksgesicht @CutestNekoAqua If you like, join yourselves in as maintainers. Certainly better to have a bus-factor > 1 for this package I suppose.
(I'll soon have a suitable device again too).

Reply:
@Keksgesicht As per #343483 -> https://github.com/NixOS/nixpkgs/blob/2c799a1db3bcc5b1c77cb127d219a1fe8fee864d/pkgs/os-specific/linux/tuxedo-drivers/default.nix#L43

Also getting @xaverdh into the loop. So they know we're trying to tighten up all the various loose ends.

Reply:
I am not sure, whether I should add other people. But as they clearly want to develop this further, I could add them in the commit too.

Reply:
feel free to add me!